src/pages/contact/index.module.scss

```diff
@@ -0,0 +1,6 @@
+@import "~bootstrap/scss/_functions.scss";
+@import "~bootstrap/scss/_variables.scss";
+
+.container {
+  background-color: $gray-200;
```

Comment:
It feels a little jarring to end up on a page with a slightly different background color than the rest of the site - can you explain the rationale behind changing this?

Reply:
I was thinking of using gray and white to separate containers in pages with a lot of information (for example, main page and gallery). Agree that it does not make a difference in the Contact page. I will leave this as one of the design open questions that we can discuss next meeting.